packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -159,79 +160,116 @@ const SimpleStatistic: FC<SimpleStatisticsProps> = ({
     if (isLoading) return <LoadingChart />;
 
     return validData ? (
-        <BigNumberContainer
-            $paddingBottom={
-                isDashboard && isTitleHidden ? 0 : TILE_HEADER_HEIGHT
-            }
+        <Center
+            w="100%"
+            component={Stack}
+            spacing={0}
+            pb={isDashboard && isTitleHidden ? 0 : TILE_HEADER_HEIGHT}
             ref={(elem) => setRef(elem)}
             {...wrapperProps}
         >
-            <BigNumberHalf>
+            <Flex style={{ flexShrink: 1 }}>
                 {minimal || isSqlRunner ? (
-                    <BigNumber $fontSize={valueFontSize}>{bigNumber}</BigNumber>
+                    <Text
+                        fz={valueFontSize}
+                        c="dark.4"
+                        align="center"
+                        fw={500}
+                        style={{
+                            transition: 'font-size 0.1s ease-in-out',
+                        }}
+                    >
+                        {bigNumber}
+                    </Text>
```

Comment:
these `<Text` components repeat a lot. maybe it's better to create wrapper components?

Reply:
done.